### PR TITLE
enforce mime types for endpoints

### PR DESCRIPTION
**Pull request by ymhari**
# PR Checklist

Please check if your PR fulfills the following requirements
- [X] The title follows our [guidelines](https://github.com/ideacrew/enroll/blob/trunk/CONTRIBUTING.md#commit)
- [X] Tests for the changes have been added (for bugfixes/features), they use let helpers and before blocks
- [ ] For all UI changes, there is cucumber coverage
- [ ] Any endpoint touched in the PR has an appropriate Pundit policy. For open endpoints, reasoning is documented in PR and code
- [ ] For all scripts or rake tasks, how to run it is documented on both the PR and in the code
- [ ] There are no inline styles added
- [ ] There are no inline javascript added
- [ ] There is no hard coded text added/updated in helpers/views/Javascript. New/updated translation strings do not include markup/styles, unless there is supporting documentation
- [ ] Code does not use .html_safe
- [ ] All images added/updated have alt text
- [X] Doesn’t bypass rubocop rules in any way

# PR Type
What kind of change does this PR introduce?

- [X] Bugfix
- [ ] Feature (requires Feature flag)
- [ ] Data fix or migration (inert code, no impact until run)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Dependency updates (e.g., add a new gem or update to a version)

# What is the ticket # detailing the issue?

Ticket: https://www.pivotaltracker.com/story/show/187465810

# A brief description of the changes

Current behavior: does not enforce mime type for some endpoints

New behavior: enforcing mime type for missing endpoints

# Feature Flag

For all new feature development, a feature flag is required to control the exposure of the feature to our end users. A feature flag needs a corresponding environment variable to initialize the state of the flag. Please share the name of the environment variable below that would enable/disable the feature and which client(s) it applies to.

Variable name:

- [ ] DC
- [ ] ME

# Additional Context
Include any additional context that may be relevant to the peer review process.

# AppScan CodeSweep Failure
In the event of a failed check on the AppScan CodeSweep step of our GitHub Actions workflow, please review the False Positive protocol outlined here: appscan_codesweep/CODESWEEP_FALSE_POSITIVES_README.MD

Add all required notes to this section if the failure is a suspected false positive.
Note: There seems to be false positive for CrossSiteRequestForgery. We are already using `protect_from_forgery` in our application_controller which should prevent this issue.

### CODE REVIEW

File: app/controllers/insured/consumer_roles_controller.rb [hunk 1]
```diff
@@ -274,12 +277,18 @@ def ridp_agreement
...
+    end
   end
 
   def upload_ridp_document
     set_consumer_bookmark_url
     set_current_person
     @person.consumer_role.move_identity_documents_to_outstanding
+    respond_to do |format|
+      format.html
+    end
```
Comment (TristanB17) [inline app/controllers/insured/consumer_roles_controller.rb:291]: I believe if there's not a specific target for the `render` action, this respond_to block can be written as 
```
respond_to :html
```

**Review by TristanB17 — DISMISSED** [review]
LGTM! Just added a single optional comment, not necessary to update PR

**Review by vkghub — CHANGES_REQUESTED** [review]
One more method is missing 
- https://github.com/ideacrew/enroll/pull/3770/files#diff-2a0aa5303b979e3a8f8e489305be1c81a26dfb6538b8510cf1ac4e25e62a2401R145
- build_insured_consumer_role_index_path

**Review by desireB — APPROVED** [review]
I confirm that protect_from_forgery is present in https://github.com/ideacrew/enroll/blob/trunk/app/controllers/application_controller.rb L24, with no exception. This is indeed a false positive.

**Review by vkghub — APPROVED** [review]

**Review by TristanB17 — APPROVED** [review]

**Pull request merged**